Tomcat: Make JMX metrics available on both standalone and embedded Tomcat versions (#15761)
DataDog/integrations-core: tomcat/README.md

## Troubleshooting

### Missing `tomcat.*` metrics

> The most recent version of the Datadog Agent will collect metrics from JMX at either `Catalina` or `Tomcat` as bean domain names.

Reviewer: ```suggestion
The Datadog agent collects JMX metrics with either `Catalina` or `Tomcat` as bean domain names.
```

Author: Change applied.

> Regular standalone Tomcat deployments have those metrics under domain `Catalina`, but embedded Tomcat deployments (such as with Spring Boot) have those metrics under domain `Tomcat`.

Reviewer: ```suggestion
Standalone Tomcat deployments have metrics under domain `Catalina`, but embedded Tomcat deployments (such as with Spring Boot) have metrics under domain `Tomcat`.
```

Author: Change applied.

> If your Datadog agent is older than **7.47.0**, and if the exposed Tomcat metrics are prefixed with a different bean domain name, such as `Tomcat`, copy the default metrics from the `metrics.yaml` to the `conf` section of the `tomcat.d/conf.yaml` and modify the `domain` filter to use the applicable bean domain name. 

Reviewer: ```suggestion
If the Datadog agent is older than **7.47.0**, and if the exposed Tomcat metrics are prefixed with a different bean domain name, such as `Tomcat`, copy the default metrics from the `metrics.yaml` file to the `conf` section of the `tomcat.d/conf.yaml` file and modify the `domain` filter to use the applicable bean domain name. 
```

Author: Change applied.
